requirements.txt

```
@@ -3,6 +3,7 @@ sphinx
 myst-nb
 sphinx-book-theme
 sphinx-design
+sphinx-thebe
```

Add the limit here, too (though this is old enough of a release that I'm sure it's OK from out of the box, too).

```suggestion
sphinx-thebe>=0.3.1
```